mlir/lib/Dialect/Rock/Pipelines/Pipelines.cpp

```diff
@@ -81,6 +83,8 @@ void rock::buildBufferizePipeline(OpPassManager &pm,
 
   pm.addPass(createConvertTensorToLinalgPass());
   pm.addNestedPass<func::FuncOp>(createLinalgInitTensorToAllocTensorPass());
+  pm.addNestedPass<func::FuncOp>(createLinalgFoldUnitExtentDimsPass());
+  pm.addNestedPass<func::FuncOp>(memref::createFoldMemRefAliasOpsPass());
```

Comment:
Huh, what's this pass do?

Reply:
Ah not needed really.. this was a remnant from the earlier approach to the problem.